library/cartridge_apply_app_config.py

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env python
+
+import json
+import os
+import zipfile
+
+from ansible.module_utils.helpers import Helpers as helpers
+from ansible.module_utils.urls import open_url, basic_auth_header
+
+argument_spec = {
+    'local_config_path': {'required': True, 'type': 'str'},
+    'remote_dir': {'required': False, 'type': 'str', 'default': '/tmp/'},
+    'upload_endpoint': {'required': False, 'type': 'str'},
+    'console_sock': {'required': False, 'type': 'str'},
+    'http_port': {'required': False, 'type': 'int'},
+    'cluster_cookie': {'required': False, 'type': 'str'},
+    'tdg_token': {'required': False, 'type': 'str'},
+}
+
+LUA_ENDPOINT = 'lua'
+HTTP_ENDPOINT = 'http'
+TDG_ENDPOINT = 'tdg'
+
+PURE_DIR_MODE = 'pure'
+ZIP_DIR_MODE = 'zip'
+
+
+def patch_file_clusterwide(control_console, file_path):
+    file_ext = os.path.splitext(file_path)[1]
+    assert file_ext in ['.yml', '.yaml'], "Impossible to use '%s' file in patch clusterwide function" % file_ext
+
+    new_sections, err = helpers.read_yaml_file(control_console, file_path)
+    assert err is None, err
+
+    old_sections, err = helpers.get_clusterwide_config(control_console)
+    assert err is None, err
+
+    for section, value in old_sections.items():
+        new_sections[section] = new_sections.get(section)
```

Comment:
```python
new_sections.update({
    name: new_sections.get(section)
    for section in old_sections
})
```

What if some section were specified in old config, but wasn't in new? Is it ok to simply remove it?

Reply:
Yes, because it's a standard behavior of HTTP endpoint